File: PHP/mysql/pro7.php
<?php
    /*
       alter table
    */ 

    // create required variables
    $hostname = "localhost";
    $username = "root";
    $password = "";
    $database = "mydb";

    try{
        // create connection with MySql Server
        $connection = new mysqli($hostname, $username, $password, $database);

        // check for connection error 
        if($connection->connect_error){
            throw new Exception("Connection Filed ".$connection->connect_error);
        }else{
            // query for table create
            $sqlquery = "alter table students add column state varchar(20)not null after city";

            // execute a query
            if($connection->query($sqlquery) === true){
                echo "<hr> Table Updated";
            }else{
                throw new Exception("Error while updating new Table");
            }
        }
    }catch(Exception $err){
        echo "<hr>Error is ".$err->getMessage();
    }finally{
        $connection->close();
    }
?>

File: Python/mySql/pro7.py
from sys import prefix

import mysql.connector

try:
    connection = mysql.connector.connect(host="localhost", user="root",
                                         passwd="", database="pythondb")
    cursor = connection.cursor()
    query = ("insert into students (fname, lname, city) values (%s, %s, %s)")
    data = ["Udit", "Ghetiya", "Rajkot"]
    cursor.execute(query, data)
    print("New Record Created")
except Exception as err:
    print("Error is ",err)
finally:
    connection.close()